buildSrc/src/main/groovy/com/uber/okbuck/experimental/bazel/README.md

```diff
@@ -0,0 +1,77 @@
+# OkBazel - Experimental Bazel Support for OkBuck
+
+[Bazel](https://bazel.build) is an [open source]
+(https://github.com/bazelbuild/bazel) build system made by Google for
+fast, correct and reproducible builds. While Bazel supports many different
+languages, OkBazel targets only server Java and Android applications.
+Specifically, it transforms applications of the Gradle `java` and `android`
+plugins into Bazel `java_library`, `java_binary`, `android_library` and
+`android_binary` targets.
+
+## Running OkBazel
+
+First, ensure that the OkBuck jar is in the classpath of your Gradle
+dependencies just as you would to run OkBuck. Then to apply the plugin, instead
+of
+
+    apply plugin: 'com.uber.okbuck'
+
+put the following in your `build.gradle`:
+
+    apply plugin: 'com.uber.okbazel'
+
+Then to run OkBazel which will generate the Bazel `BUILD` files, from the root
+of your Gradle project run
+
+    ./gradlew okbazel
+
+This will generate one `BUILD` file for each Gradle subproject, one `BUILD` file
+in `okbazel/BUILD` and one `WORKSPACE` file. The `WORKSPACE` file contains an
+`android_sdk_repository` rule which will contain the path of your Android SDK if
+the `$ANDROID_HOME` environment variable is set. You will need to update
+`WORKSPACE` to contain the `api_level` and `build_tools_version` from your SDK
+that you wish to use.
+
+## Setting up Bazel
```

Comment:
I will look into how `buckw` works and respond back on this thread. However, I don't think installing bazel from source is a possibility. Bazel can only be built with a pre-existing installation of Bazel. We do this to maintain platform independence (e.g. we don't want to be shipping pre-compiled, platform-specific binaries in the Bazel source tree that would be needed to bootstrap from scratch).

Furthermore, Bazel is very carefully versioned and, since it is pre-1.0, occasionally makes breaking changes. As such, we have some users who intentionally do not update to the newest version immediately because it may require a change to their `BUILD` files. We keep a history of platform-specific installers [here](https://github.com/bazelbuild/bazel/releases) and a detailed changelog for each release [here](https://github.com/bazelbuild/bazel/blob/master/CHANGELOG.md).

I agree that re-running okbuck when files have changed is important for usability.